### PR TITLE
Feature - Updating composer to use forked activerecord dependency

### PR DESCRIPTION
**Pull request by jstruzik**
Times, they are a changing.

I've made a new branch called `omr-dev`. This branch will make use of the forked [`robinpowered/php-activerecord`](https://github.com/robinpowered/php-activerecord) repo so that we may begin to actually override some of the functionality of ActiveRecord since we all know nobody is committing to the original repo anymore.

The `robinpowered/php-activerecord` is set to the `v1.2.0` release with a composer name update on top.

We can start to do new releases off of the `omr-dev` branch (that is, if we ever really have to update this repo again).

### CODE REVIEW

**Comment by brianmuse** [comment]
Why to OMR? I think I'd pref this on RPI for various stupid legal reasons.

**Comment by jstruzik** [comment]
Then we should migrate this repo as well.

**Comment by brianmuse** [comment]
Yeah that's what I mean. Is anything using it? Can transfer or maybe we just fork?

**Comment by brianmuse** [comment]
We maybe should eventually, but one thing at a time. Since we're adding the fork of AR to composer now, it's a dec time for that.

**Comment by jstruzik** [comment]
Yeah I agree. Should we remove the onemightyroar fork then? I don't think anything is using it.

**Comment by Rican7** [comment]
Yea, we already have the OMR fork. We could "move" (remove the old and add a new) fork to RPI.

**Comment by brianmuse** [comment]
Let's leave the OMR fork, just in case, but refork to Robin?

**Comment by jstruzik** [comment]
Sounds good.

**Comment by jstruzik** [comment]
Oops haha I can just update this PR.

**Comment by jstruzik** [comment]
Should be good now!

File: composer.json [hunk 1]
```diff
@@ -17,9 +17,15 @@
...
+  "repositories": [
+    {
+      "type": "git",
+      "url":  "git@github.com:robinpowered/php-activerecord.git"
+    }
+  ],
    "require": {
       "php": ">=5.3.0",
-      "php-activerecord/php-activerecord": "1.1.x"
+      "onemightyroar/php-activerecord": "~1.1.0"
```
Comment (brianmuse) [inline composer.json]: why still omr?

Comment (jstruzik) [inline composer.json]: oops!

Comment (brianmuse) [inline composer.json]: i think youve lost your mind

File: composer.json [hunk 2]
```diff
@@ -17,12 +17,18 @@
...
    },
    "suggest": {
-      "onemightyroar/predis-toolkit": "To make use of the AbstractRedisIntegrationModel"
+      "robinpowered/predis-toolkit": "To make use of the AbstractRedisIntegrationModel"
```
Comment (Rican7) [inline composer.json]: I think you replaced the wrong one here. :stuck_out_tongue_closed_eyes:

Comment (jstruzik) [inline composer.json]: You guys are crazy!! What are you talking about ???????

**Comment by brianmuse** [comment]
It was a long and hard road for this very complicated PR, but looks GTM :+1:

**Comment by Rican7** [comment]
wrestling-referee-thumbs-up


:+1:

**Pull request merged**